Starcraft2Client.py

```diff
@@ -3,7 +3,7 @@
 import ModuleUpdate
 ModuleUpdate.update()
 
-from worlds.sc2.Client import launch
+from worlds.sc2.client import launch
```

Comment:
We're moving away from client .exe's entirely. The client should be moved into the Launcher.